### PR TITLE
Feature/hotels list with particular hotel view

### PR DESCRIPTION
**Pull request by mikevelko**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -63,6 +64,7 @@ function App() {
         ></Route>
         <PrivateRoute authed={isUserAuthenticated} exact path='/home' component={MainPage} />
         <PrivateRoute authed={token} exact path='/hotels' component={Hotels} />
+        <PrivateRoute authed={token} exact path='/hotels/:hotelId' component={Hotel} />
         <PrivateRoute authed={isUserAuthenticated} exact path='/client' component={Client} />
         <PrivateRoute authed={isUserAuthenticated} exact path='/reservations' component={MyReservations} />
```
Comment (FlowerAlex) [inline client/src/App.js:69]: Should 'authed' depend on 2 variables? Can u combine it?

File: client/src/components/Hotel.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    const classes = useStyles();
+    return (
+
+        <div className="hotel-container">
+            <div className="hotel-container-item">
+                <p>{hotel.hotelName}</p>
+                <p>{hotel.hotelDesc}</p>
+                <p>{hotel.city}, {hotel.country}</p>
+                <Button variant="contained" color="primary" style={{ fontSize: '42px', maxWidth: '30%', maxHeight: '100px', minWidth: '30%', minHeight: '100px' }}
```
Comment (FlowerAlex) [inline client/src/components/Hotel.jsx:103]: move it to css or use material-ui function useStyles

Comment (mikevelko) [reply client/src/components/Hotel.jsx:103]: no sense to use useStyles here, will be moved to css later, thanks

File: client/src/App.js (hunk 1 above)
Comment (mikevelko) [reply client/src/App.js:69]: isUserAuthenticated is just testing thing, check new functionality about hotels please :)

**Review by FlowerAlex — APPROVED** [review]
Ok

**Pull request merged**